### ISSUE TITLE
[BUG] Incorrect parameter highlighting when commas appear not as parameter separators

### ISSUE THREAD

**Issue by markovejnovic** [markovejnovic/vim-dssl2#8]
**To Reproduce**
```
def heap_sort[X](v: VecC[X], lt?: FunC[X, X, bool?]) -> NoneC:
```
will not get highlighted correctly.

**Information (please complete the following information):**
 - OS: Arch Latest
 - nVim

**Comment by markovejnovic** [comment]
![image](https://user-images.githubusercontent.com/15203893/98156743-7f278e00-1ed8-11eb-9297-a715f06f2686.png)

**Comment by markovejnovic** [comment]
Resolved in d60d0e4
